config/default-config.json

```diff
@@ -13,6 +13,12 @@
             "allow insecure transport": "false"
         }
     },
+    "ip filtering" : {
+	    "allowPrivateIPAddress": false,
+	    "allowMetaIPAddress": false,
+	    "allowIPAddressList": "",
```

Comment:
I think `[]` is a better default value for the last two, avoiding misunderstanding (would also require update in tests).